Use latest YASGUI directly instead of YASQE/YASR

kinow opened the pull request:
GitHub issue resolved #1334

Pull request Description:

Uses YASGUI that supports tabs, instead of YASQE + YASR (used by YASGUI).

----

 - [ ] Tests are included.
 - [ ] Documentation change and updates are provided for the [Apache Jena website](https://github.com/apache/jena-site/)
 - [ ] Commits have been squashed to remove intermediate development commit messages.
 - [ ] Key commit messages start with the issue number (GH-xxxx or JENA-xxxx)

By submitting this pull request, I acknowledge that I am making a contribution to the Apache Software Foundation under the terms and conditions of the [Contributor's Agreement](https://www.apache.org/licenses/contributor-agreements.html).

----

See the [Apache Jena "Contributing" guide](https://github.com/apache/jena/blob/main/CONTRIBUTING.md).

kinow commented:
This one is still a long way before being merged. Needs more testing and better integration Vue.js & YASGUI (those tabs are tricky to get working well).

afs commented:
jena-fuseki-ui now uses `"@zazuko/yasqe": "^4.2.29"` and `"@zazuko/yasr": "^4.2.30`.

These artifacts are bug-fixed versions of `@triply/yasgui`. Hopefully, `@zazuko/yasgui` v4.2.30 works for this PR.

kinow commented:
Next on my list, I think.

kinow commented:
rebased, adjusted a bit the Vue code, will push what I have for now, even though there are some errors it's possible to preview what YASGUI looks like with the tabs.

![image](https://github.com/apache/jena/assets/304786/3ba9d30c-45a7-447b-9da2-ed115fdcc13a)

I want to compare it with Lumino (both at complexity to maintain and features -- with Lumino you organize your tabs however you want).

https://github.com/tupilabs/vue-lumino/ (disclaimer, I wrote this after using that in a previous project).
